docs/topics/proc_show-debug-logging-openshift-nodejs.adoc

```diff
@@ -0,0 +1,88 @@
+= Show Debug Logging on OpenShift
+
+Use the the `DEBUG` environment variable in your application pod in OpenShift to enable debug logging.
+
+.Prerequisites 
+* An application with debug logging.
+* The `oc` CLI client installed.
+
+
+.Procedure
+
+. Use the `oc` CLI client to log into your OpenShift instance.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Deploy your application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm run openshift
+----
++
+This runs the `openshift` npm script, which wraps direct calls to xref:about-nodeshift[nodeshift].
+
+. Find the name of your pod and follow the logs to watch it start.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get pods
+....
+$ oc logs -f pod/POD_NAME
+----
++
+IMPORTANT: After your pod has started, keep watching the logs.
+
+. Test your application. 
++
+For example, if you had debug logging in the xref:mission-http-api-nodejs[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. Return to your pod logs and notice there are no debug logging messages in the logs. 
+
+. Set the `DEBUG` environment variable in your application pod to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc set env pod/POD_NAME DEBUG=mybooster
```

Comment:
Does this work? It didn't for me, which is why the Java-based runtimes debugging docs (which also need to set an environment variable) change the entire deployment config. Changing a single pod would be much nicer! I need to confirm.

Reply:
I believe it should work, yes, since a config change like setting an env variable will cause a new pod to spin up and the only thing that determines whether debugging output is shown is that env var.